components/aws/sagemaker/tests/integration_tests/component_tests/test_hpo_component.py

```diff
@@ -7,7 +7,6 @@
 from utils import minio_utils
 from utils import sagemaker_utils
 
-
```

Comment:
can you run black command on integration_test directory, unrelated changes are showing up